src/common/snippets/src/lowered/pass/serialize_data_flow.cpp

```diff
@@ -42,6 +42,7 @@ bool SerializeDataFlow::run(const LinearIR& linear_ir) {
             results.push_back(result);
         } else {
             const auto serialization_node = std::make_shared<op::SerializationNode>(inputs, expr, serialization_mode);
+            serialization_node->get_rt_info()["execTimeMcs"] = 0;
```

Comment:
Can we mark only Parameter node (as it is done in control flow serialization)? To avoid unnecessary info serialization for each node

Reply:
Sure, done